index.html

```diff
@@ -54,20 +54,20 @@
 
 <p>The Gregorio project offers tools for typesetting Gregorian chant. These tools include:</p>
 <ul>
-<li><a href="/gabc/">gabc</a>: a notation for representing Gregorian chant using ASCII characters</li>
-<li><a href="/gregoriotex/">GregorioTeX</a>: a TeX style for typesetting scores</li>
-<li>a software application to convert from gabc to GregorioTeX</li>
+<li><a href="./gabc/">gabc</a>: a notation for representing Gregorian chant using ASCII characters</li>
+<li><a href="./gregoriotex/">Gregorio<span class="tex">T<span class="epsilon">e</span>X</span></a>: a <span class="tex">T<span class="epsilon">e</span>X</span> style for typesetting scores</li>
+<li>a software application to convert from gabc to Gregorio<span class="tex">T<span class="epsilon">e</span>X</span></li>
 </ul>
 
-<p>Together, these tools, added to a <a href="/gregoriotex/tex.html">TeX</a> installation, allow the user to engrave beautiful Gregorian chant scores.</p>
+<p>Together, these tools, added to a <a href="./gregoriotex/tex.html"><span class="tex">T<span class="epsilon">e</span>X</span></a> installation, allow the user to engrave beautiful Gregorian chant scores.</p>
 
 <div class="legende">
-<img src="illus/Alleluia.png" title="" alt="" />
+<img src="./illus/Alleluia.png" title="" alt="" />
 </div>
 
 <p>The Gregorio project consists of 100% <a href="https://en.wikipedia.org/wiki/Free_software">free software</a>.</p>
```

Comment:
I would make mention of GPLv3 here (i.e., ... free software licensed under the GPLv3 or newer) and link to that license as well.